readme.md

```diff
@@ -51,3 +51,5 @@ You will need to set your lambda's trigger to be S3 ObjectCreated.
 | BPM_USER          | Username for accessing BPM |
 | BPM_PW            | Password for above |
 | ATTACHMENT_BUCKET | S3 bucket name to store attachments in |
+
+When terraforming, you will need to set environment variables `TF_VAR_BPM_USER` and `TF_VAR_BPM_PW` to the values you have created for your service account in BAW.
```

Comment:
Why does the terraform use different variables?

Reply:
Terraform refers to a number of environment variables to customize various aspects of its behaviour. Environment variables can be used to set variables. The environment variables must be in the format TF_VAR_name and this will be checked last for a value. - https://www.terraform.io/docs/commands/environment-variables.html